src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -6974,11 +6967,7 @@ extends System.Object
 public record B : A {
 }";
             var comp = CreateCompilationWithIL(new[] { source, IsExternalInitTypeDefinition }, ilSource: ilSource, parseOptions: TestOptions.Regular9);
-            comp.VerifyEmitDiagnostics(
-                // (2,15): error CS0239: 'B.ToString()': cannot override inherited member 'A.ToString()' because it is sealed
-                // public record B : A {
-                Diagnostic(ErrorCode.ERR_CantOverrideSealed, "B").WithArguments("B.ToString()", "A.ToString()").WithLocation(2, 15)
```

Comment:
We no longer try to generate ToString() if the base implementation is sealed, so this diagnostic disappears.